src/snowcli/cli/__init__.py

```diff
@@ -33,101 +31,6 @@ def version_callback(value: bool):
         raise typer.Exit()
 
 
-@app.command()
-def login(
```

Comment:
1. Do I correctly understand that now we won't have to login and configure connection and instead of this we will have to specify connection for each command using it if we don't want to use "default" connection?
2. If that's true, then IMO we should update README.

Reply:
That's correct. I will update the README